R/quanteda_options.R

```diff
@@ -129,13 +129,13 @@ set_option_value <- function(key, value) {
             warning("Setting threads instead to maximum available ", thread["max"], call. = FALSE)
             value <- thread["max"]
         }
-        if (is.na(thread["tbb"])) {
+        # if (is.na(thread["tbb"])) {
```

Comment:
Why did we need this condition to be TRUE before we call `RcppParallel::setThreadOptions(value)`? Because it can only be set once, and if not `NA`, then it has already been set?